template/LICENSE.jinja

```diff
@@ -0,0 +1,6579 @@
+[% from pathjoin("includes", "variable.jinja") import repo_url with context -%]
```

Comment:
1. Seems there are some fields not formatted in jinja variables. I know there are too many of them, that is why I suggest to maintain only the most popular ones.
2. Instead of having a huge file, I would suggest to have each license in separated files in the `includes/licenses/` folder so that we can include them here. In this way, it will be straightforward if we want to support more licenses in the future. Note that we only need templates rather than macros. You can find the reference [here](https://copier.readthedocs.io/en/stable/configuring/#importing-jinja-templates-and-macros).